src/Azure.Functions.Cli/Common/Utilities.cs

```diff
@@ -205,12 +205,14 @@ internal static bool LogLevelExists(IConfigurationRoot hostJsonConfig, string ca
             string categoryKey = ConfigurationPath.Combine(ConfigurationSectionNames.JobHost, ConfigurationSectionNames.Logging, LogLevelSection, category);
             try
             {
-                if (Enum.TryParse(typeof(LogLevel), hostJsonConfig[categoryKey].ToString(), true, out object outLevel))
+                if (hostJsonConfig[categoryKey] != null && Enum.TryParse(typeof(LogLevel), hostJsonConfig[categoryKey].ToString(), true, out object outLevel))
```

Comment:
What if you just removed the `.ToString()`? I assume that's where the null-ref was? That should already be a string and I think `Enum.TryParse()` can handle that.

Reply:
thanks. you are right. I assumed `Enum.TryParse` would throw on if the input was `null`